data/scripts/spells/support/levitate.lua

```diff
@@ -23,11 +31,10 @@ function spell.onCastSpell(creature, variant)
 	local returnValue = levitate(creature, variant:getString():lower())
 	if returnValue ~= RETURNVALUE_NOERROR then
 		creature:sendCancelMessage(returnValue)
-		creature:getPosition():sendMagicEffect(CONST_ME_POFF)
-		return false
-	end
 
-	creature:getPosition():sendMagicEffect(CONST_ME_TELEPORT)
+creature:getPosition():sendMagicEffect(CONST_ME_POFF)
+	return false
+end
 	return true
 end
```

Comment:
Please fix the indentation:
```suggestion
		creature:getPosition():sendMagicEffect(CONST_ME_POFF)
		return false
	end
	return true
end
```